### PR TITLE
test: Fix Python Docstring to include all Args.

### PR DESCRIPTION
**Pull request by jbampton**
Found a Python function that had incorrect and missing arguments in its Docstring.

### CODE REVIEW

**Comment by MarcoFalke** [comment]
Might as well insert a `, *,` before `timeout` in the function signature to enforce named args?

File: test/functional/test_framework/util.py [hunk 1]
```diff
@@ -244,14 +244,13 @@ class PortSeed:
...
     n = None
 
-def get_rpc_proxy(url, node_number, timeout=None, coveragedir=None):
+def get_rpc_proxy(url, node_number, *, timeout=None, coveragedir=None):
     """
     Args:
         url (str): URL of the RPC server to call
         node_number (int): the node number (or id) that this calls to
-
-    Kwargs:
```
Comment (laanwj) [inline test/functional/test_framework/util.py]: Any reason to remove "Kwargs" here? Especially with the `*, ` it seems important to mention that these need to be keyword arguments.

Comment (jbampton) [reply test/functional/test_framework/util.py]: I put `Kwargs:` back now.

**Comment by laanwj** [comment]
ACK 8acd58927a614e006641384f61f8e7fca1cc68fc

**Pull request merged**